android/app/build.gradle

```diff
@@ -212,8 +212,8 @@ android {
             applicationId 'org.pathcheck.covidsafepaths'
             dimension "version"
         }
-        bte {
-            applicationId 'org.pathcheck.covidsafepathsBte'
+        bt {
+            applicationId 'org.pathcheck.covidsafepathsBt'
```

Comment:
here too - we should keep the appId in `package.json` consistent with gradle

Reply:
Changed - thanks!